src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/filters/MdcLogEnhancerFilter.kt

```diff
@@ -13,11 +13,13 @@ class MdcLogEnhancerFilter : OncePerRequestFilter() {
     // this is the place to add more request specific log fields...
 
     MDC.put("hostname", req.localAddr)
+    MDC.put("req_id", req.getHeader("X-Request-Id"))
```

Comment:
Can we use full names, please? I.e. `request_id`

Reply:
i requested this naming to be consistent with some standard log fields in the UI. see https://github.com/trentm/node-bunyan#recommendedbest-practice-fields for more details on these.

Reply:
thanks, TIL